WordPress/Classes/Extensions/NSParagraphStyle+Helpers.swift

```diff
@@ -1,15 +1,15 @@
 import Foundation
 
-
-extension NSMutableParagraphStyle {
-    @objc convenience init(minLineHeight: CGFloat, lineBreakMode: NSLineBreakMode, alignment: NSTextAlignment) {
+@objc
```

Comment:
I don't think this needs to be `@objc`.

Reply:
Both of the methods in this extension were [also annotated](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/Extensions/NSParagraphStyle%2BHelpers.swift) as `@objc`.

I'll assume that the same change we agree upon above should be applied here.